src/widget/wspinny.cpp

```diff
@@ -176,7 +178,19 @@ void WSpinny::setup(const QDomNode& node, const SkinContext& context) {
                 size(), Qt::KeepAspectRatio, Qt::SmoothTransformation);
     }
 
-    m_bShowCover = context.selectBool(node, "ShowCover", false);
+    // Dynamic skin option, set in WSpinny's <ShowCoverControl> node.
+    if (showCoverConfigKey.isValid()) {
```

Comment:
Does the config key for showing cover art have to be configurable? Why can't it just be a single CO defined inside Mixxx?  (e.g, always [Skin],show_coverart).  Its value never varies in the updates you made

Reply:
I'd like this to be configurable because spinny and cover art can also be shown separately, like in Deere (which I'll adjust in a separate PR): spinnies next to the waveforms always show the cover while the cover can be toggled in the decks.

Reply:
ok I see